packages/renderer/src/stores/navigation/navigation-registry-kubernetes.svelte.ts

```diff
@@ -28,6 +32,8 @@ import type { NavigationRegistryEntry } from './navigation-registry';
 
 let kubernetesNavigationGroupItems: NavigationRegistryEntry[] = $state([]);
 
+export let unsubscribe: Unsubscriber;
```

Comment:
called on unit tests (afterEach), to be able to call the function several times without conflicts

Reply:
This feel weird :thinking:

Reply:
Yes. The pattern used in the files in `packages/renderer/src/stores/navigation` does not expect to unsubscribe all the subscriptions (and because there is always 1 test for each file there, the problem was not raised before).

If we want to use a cleaner pattern (using a class extending Disposable for example), this will need to refactor all the files in this directory

Reply:
-> https://github.com/containers/podman-desktop/issues/9089

Reply:
not sure I follow everything but we can't add unsubscribe in the onDestroy() method instead of calling it in afterEach in the test (and having to export it) ?

Reply:
I'm afraid we're in the blurry part as we are in .ts file which are fragment of code, etc.